Mixpanel/app_extension_module.modulemap

```diff
@@ -1,5 +1,5 @@
 framework module MixpanelAppExtension {
-    umbrella header "Mixpanel.h"
+    header "Mixpanel.h"
```

Comment:
from what i understood umbrella headers are supposed to be headers that import all other headers. so if `Mixpanel.h` imports `MixpanelPeople.h`, then it should be the same. but if that resolves weird warnings then it seems worthy of the change. Also I think umbrella headers needs to be of the same name of the framework, so due to having different modulemaps it will probably make things more cumbersome.